README.md

```diff
@@ -2,55 +2,55 @@
 
 # Project Name
 
-Description the project.
+<<<<<<< HEAD
+- This is an application that displays a list of Covid Data generated for the african continent from an API.
 
+## Video Presentation
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+[Presentation](https://www.loom.com/share/bb3c405e43ce48228658dd2b115c6e24)
 
-## Live Demo (if available)
+=======
+This is an application that displays a list of Covid Data generated for the african continent from an API.
+>>>>>>> af4d69a1e3c47eece0acda1b4f964058da8c6da0
 
-[Live Demo Link](https://livedemo.com)
 
+## Built With
 
-## Getting Started
+- ReactJS
+- Redux
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## Live Demo
 
+[Live Demo Link](https://62b45a342c11d41099509ed1--dinma-faith-makes-great-sites.
+netlify.app/)
```

Comment:
[OPTIONAL] - Since the previous reviewer missed this, i will mark this requirement as optional, but i strongly suggest that you add a working link to your deployed app as this is a capstone project which shows mastery in react